### ISSUE TITLE
landuse timeseries for 1979-2026 for ne120 resolution

### ISSUE THREAD

**Issue by ekluzek** [ESCOMP/CTSM#2785]
CAM needs a landuse timeseries file for 1979-2026 for the ne120 resolution.

See the conversation here:

https://github.com/NCAR/amwg_dev/discussions/585#discussioncomment-10709399

**Comment by wwieder** [comment]
Since @adamrher is creating what's needed for short term tests, let's make this later, maybe even after we have CMIP7 datasets (ctsm5.4)?

**Comment by ekluzek** [comment]
@adamrher I think this should be ne120np4.pg3 instead of plain ne120np4 right? Or is there a need for both? And if so what specifically is needed for both?

**Comment by adamrher** [comment]
Correct, we only need one: ne120np4.pg3. Looking back at the old conv, it looks like folks want 1979-2030 transient support for this grid, but I noted that will depend on whether CLM agrees to store this large dataset.

**Comment by slevis-lmwg** [comment]
Marking "done" for sprint accounting because I added 1979-2030 support to the mksurfdata_esmf tool in https://github.com/ESCOMP/CTSM/pull/3482 for this grid; however, there are unchecked boxes in this issue to be addressed later.

**Comment by slevis-lmwg** [comment]
> Correct, we only need one: ne120np4.pg3. Looking back at the old conv, it looks like folks want 1979-2030 transient support for this grid, but I noted that will depend on whether CLM agrees to store this large dataset.

@adamrher your comment here explicitly requests 1979-2030; however in the [spreadsheet](https://docs.google.com/spreadsheets/d/1Osq56e423CF107zhoNQ0VS7-iH_JXLF9AtCvBdXyfJ4/edit?gid=0#gid=0) I see 1979-2023. Do you know which of the two is correct?

**Comment by adamrher** [comment]
I think I was wrong in this thread, since Erik sent us a Sept 3rd email saying that CMIP7 future emissions scenarios aren't available yet. I'm fine with 1979-2023 (or 2024). Email from @ekluzek:

> Adam, I realize I have another question and thing to point out. We don't have CMIP7 SSP data yet, so for CESM3.0 we will be pointing to the CMIP6 data when SSP compsets are used.

>This also means that we can't really do 1979-2030 datasets, as that would mix CMIP7 historical and CMIP6 future, which would have a weird discontinuity. So we'll do 1979-2024 for them. The ctsm5.3 CMIP6 1979-2026 ones are still around though which can be used to get an extra 2 years beyond. So I wanted to ask to make sure that sounded OK.

**Comment by slevis-lmwg** [comment]
Thank you @adamrher 
I forgot or missed that communication.
